detect.go

```diff
@@ -75,3 +75,13 @@ func isNodeApp(workingDir string) (bool, string, error) {
 		return false, "", err
 	}
 }
+
+func nodeWebRoot(nodeAppPath string) string {
+	// if explicitly specified, always use the WebRootEnv
+	if path, ok := os.LookupEnv(require.WebRootEnv); ok {
+		return path
```

Comment:
our env variable can be set to an empty string. Should we check it here and use the default "build" if that is the case? not sure 🙂

Reply:
Also not sure but as the webroot is always relative to the `/workspace` dir, an empty path would also be valid if you want to force the webroot to be in `/workspace`.